script/bootstrap.py

```diff
@@ -92,6 +95,18 @@ def update_node_modules(dirname, env=None):
       execute_stdout([NPM, 'install'], env)
 
 
+def update_crashpad():
+  gclient = os.path.join(VENDOR_DIR, 'depot_tools', 'gclient.py')
```

Comment:
Howerver `crashpad` repo has its own third-party libraries, which uses `gclient` tool to maintian.
In making `crashpad` a git submodule, we need to also add its third-party libraries in `crashpad` repo.